packages/phone-number-privacy/common/package.json

```diff
@@ -16,10 +16,17 @@
     "lib/**/*"
   ],
   "dependencies": {
-    "@celo/utils": "^0.1.20-dev",
-    "bignumber.js": "^9.0.0"
+    "@celo/contractkit": "0.4.14-dev",
+    "@celo/utils": "0.1.20-dev",
+    "bignumber.js": "^9.0.0",
+    "elliptic": "6.5.3",
+    "@types/elliptic": "^6.4.12",
+    "firebase-functions": "^3.6.0"
+  },
+  "devDependencies": {
+    "ts-node": "^8.3.0",
+    "typescript": "^3.8.3"
```

Comment:
We generally avoid putting these here and instead rely on the ones at the monorepo root package.json to ensure we're all using the same version